src/Apps/Comparables/index.tsx

```diff
@@ -0,0 +1,113 @@
+  // TODO: Currencies?
+  priceAsRange(price, multiplier = 100) {
+    const dollars = price / multiplier
+    const min = Math.floor(Math.max(0, 0.9 * dollars))
+    const max = Math.floor(1.1 * dollars)
+    return `${min}-${max}`
+  }
+  render() {
+    const { artworkID } = this.props
+
+    return (
+      <ContextConsumer>
+        {({ relayEnvironment }) => {
+          return (
+            <>
+              <QueryRenderer<ComparablesQuery>
+                environment={relayEnvironment}
+                query={graphql`
+                  query ComparablesQuery($artworkID: String!) {
+                    artwork(id: $artworkID) {
+                      artist {
+                        id
+                      }
+                      category
+                      attribution_class {
+                        id
+                        name
+                      }
+                      priceCents {
+                        min
+                      }
+                      ...GridItem_artwork
+                    }
+                  }
+                `}
+                variables={{ artworkID }}
+                render={({ props }) => {
+                  if (props) {
+                    const {
+                      category,
+                      artist,
+                      attribution_class,
+                      priceCents,
+                    } = props.artwork
+                    const medium = CategoryToMediumGeneMap[category]
+                    const price = priceCents && priceCents.min
+                    const priceRange = price ? this.priceAsRange(price) : "*-*"
+                    const attributionClass = attribution_class && [
+                      attribution_class.id,
+                    ]
+                    return (
+                      <StateProvider
+                        inject={[
+                          new FilterState({
+                            attribution_class: attributionClass,
+                            price_range: priceRange,
+                            medium,
+                          }),
+                        ]}
+                      >
+                        <Box pt={3} pb={3}>
+                          Viewing {artworkID} comparables
+                          <div style={{ margin: "auto", width: "200px" }}>
+                            <RelayGridItem artwork={props.artwork} />
+                          </div>
+                          <Comparables
+                            attributionClass={attributionClass}
+                            artistID={artist.id}
+                            medium={medium}
+                            priceRange={priceRange}
+                          />
+                        </Box>
+                      </StateProvider>
+                    )
+                  } else {
+                    return (
+                      <SpinnerContainer>
+                        <Spinner />
+                      </SpinnerContainer>
```

Comment:
Can use this recent util: https://github.com/artsy/reaction/blob/master/src/Artsy/Relay/renderWithLoadProgress.tsx